### PR TITLE
Multiple items in loadouts

### PR DESCRIPTION
**Pull request by TheShuEd**
## About the PR
Added the ability to create loadouts from a number of items greater than 1.

Until now, if more than 1 item is selected, the loadoout would give an “error” without showing images or names. Now it is possible to override dummyEntity, from which the name, description and sprite are taken. 

To give you an example, a three-dart loadout has been added. This also reduces the hypodart metagame a bit.

## Media
image
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Changelog
🆑 
- add: You can now select candles in trinkets loadout

### CODE REVIEW

**Review by metalgearsloth — APPROVED** [review]

**Comment by metalgearsloth** [comment]
Needs cl.

**Comment by slarticodefast** [comment]
The code looks good and works as intended, but before merging I would replace the example item with something different. The three darts can each be used to transfer 2u of a reagent into a target and I think loadouts should not contain any weapons.

**Comment by TheShuEd** [comment]
image
image

File: Resources/Prototypes/Loadouts/dummy_entities.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- type: entity
+  id: LoadoutDummyCandles
+  categories: [ HideSpawnMenu ]
+  name: three candles
+  description: A set of three colorful candles to secret rituals!
```
Comment (slarticodefast) [inline Resources/Prototypes/Loadouts/dummy_entities.yml:5]:
```suggestion
  description: A set of three colorful candles for secret rituals!
```

**Comment by OnsenCapy** [comment]
Might cause issues in the future if heretics ever gets merged since one of their rituals requires candles

**Comment by Errant-4** [comment]
Then we can just remove candles from the loadouts at that time, if this becomes a problem

**Pull request merged**